src/main/java/com/teragrep/rlp_03/context/frame/rental/Lease.java

```diff
@@ -1,12 +1,12 @@
-package com.teragrep.rlp_03.context.frame.access;
+package com.teragrep.rlp_03.context.frame.rental;
 
 public class Lease implements AutoCloseable {
```

Comment:
could this be refactored so that it uses a sub-phaser that is registered to the parent?